eos-chrome-plugin-update-upstream

```diff
@@ -21,6 +21,21 @@ DEST_DIR=/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater
 WGET_COMMAND="wget -c --tries=5"
 DAYS_CHECK=7
 
+# Data needed for the Flash plugin, from Adobe
+FLASH_FALLBACK_VERSION="23.0.0.185"
+FLASH_FALLBACK_URL="https://fpdownload.adobe.com/pub/flashplayer/pdc/${FLASH_FALLBACK_VERSION}/flash_player_ppapi_linux.$(arch).tar.gz"
+FLASH_FALLBACK_SHA256SUM="3f7a13a615d3cf19e5feb02485e12e3785a6ef07cd7ae32072cd860468fd322b"
+
+FLASH_LKGV_FILE_URL="https://raw.githubusercontent.com/endlessm/eos-chrome-plugin-updater/master/flashplugin-$(arch).lkgv"
```

Comment:
I'm not sure about having production code pulling from a master branch.  I'm also not sure about having all users start by trying to read a file from github, though I suppose we have the fallback.  Probably worth getting the opinion of @cosimoc.